### DISCUSSION TITLE
Auto sign in after sign up

### DISCUSSION THREAD

**Discussion by Bassaamm** [supabase/supabase#15857] [Questions] [answered]
When I use the auth sign up and it return successfully. It automatically sign in the user which is a behavior don't want to it be. So the user after signing up should log in not get logged in directly. So how should I disable the auto signing in because I have search in doc and issues section.

**Comment by GaryAustin1** [comment, marked as answer]
There is no option to not sign in when signing up.  If you have Confirm Email enabled in the dashboard then the user will not be signed in until they confirm their email, but will be when that happens.  With Confirm Email disabled you are signed in immediately and a session is returned.
